src/jira/client/axios.ts

```diff
@@ -91,7 +91,7 @@ function getErrorMiddleware(logger: Logger) {
 				const errorMessage = getJiraErrorMessages(status);
 				// Creating an object that isn't of type Error as bunyan handles it differently
 				// Log appropriate level depending on status - WARN: 300-499, ERROR: everything else
-				(status >= 300 && status < 500 ? logger.warn : logger.error)({ ...error, response: error.response, resquest: error.request }, errorMessage);
+				(status >= 300 && status < 500 ? logger.warn : logger.error)(error, errorMessage);
```

Comment:
This previously didn't log the path name of the failed request:
![image](https://user-images.githubusercontent.com/1363578/131761734-e1b79a11-bb7d-40d6-9423-c8d5e3495381.png)

We should add the path so that we know which endpoint causes the errors (and all those 404s)

Reply:
It will with the latest serializer for the logger.